### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ManuSanRam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,5 +4,13 @@
 # uno-js
 
 Unosquare Typescript/JavaScript Library, zero dependencies.
+String and array manipulation, easy-to-use fetch controller and more, written in Typescript.
+
+### Installation (npm)
+To install uno-js through npm, you only need to write down the following command:
+
+    npm install uno-js --save
```
Comment (geoperez) [inline README.md]: Use code blocl

File: README.md [hunk 2]
```diff
@@ -4,5 +4,13 @@
...
+
+npm link - [uno-js package](https://www.npmjs.com/package/uno-js)
```
Comment (geoperez) [inline README.md]: Add a badge fury

Comment (ManuSanRam) [reply README.md]: which badge?

Comment (ManuSanRam) [reply README.md]: which badge?

Comment (geoperez) [reply README.md]: https://badge.fury.io/

**Pull request merged**